packages/entity-database-adapter-knex/src/PostgresEntityDatabaseAdapter.ts

```diff
@@ -67,6 +67,12 @@ export default class PostgresEntityDatabaseAdapter<TFields> extends EntityDataba
       }
     }
 
+    if (orderByRaw !== undefined) {
+      for (const orderByRawSpecification of orderByRaw) {
```

Comment:
I'm thinking that we probably don't want this to be an array, though I could be convinced otherwise. The reason is that even though knex supports it, we don't support multiple `whereRaw` in `fetchManyByRawWhereClauseInternalAsync`, so we should keep them in alignment.